### PR TITLE
arthur config

### PR DESCRIPTION
**Pull request by Vighnesh-V**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,8 @@ and [ppx_tools/rewriter](https://github.com/ocaml-ppx/ppx_tools/blob/master/rewr
 - compiler-libs.common
 - fswatch (for Build + Watch)
 - ANSITerminal
-- ppx_expect (for testing)
+- ppx_expect >= 0.13.1
```
Comment (KeenWill) [inline README.md:20]: we said below the ppx_expect is only for testing... Is this the case or should I change it to be a build dependency too?

Comment (Vighnesh-V) [reply README.md:20]: hmm. Lets leave it as build dependency. It seems to pass the checks online.

Comment (KeenWill) [reply README.md:20]: wait it's currently a test dependency i mean, but the readme says it's build dep

Comment (Vighnesh-V) [reply README.md:20]: oh gotcha. Uhh, I'll add back your change for the testing note, since it's not strictly build

Comment (KeenWill) [reply README.md]: sounds good

**Pull request merged**